build.properties.dist

```diff
@@ -87,15 +85,11 @@ behat.base_url = ${drupal.base_url}
 # The browser to use for testing, either 'firefox' or 'chrome'.
 behat.browser_name = chrome
 
-# The webdriver host.  'selenium' is the hostname of the apache docker container.
-# For local development, this should be overridden.
-behat.webdriver_host = selenium
-
 # The port of the webdriver host (e.g. Selenium or PhantomJS).
 behat.webdriver_port = 4444
 
 # The URL of the webdriver host (e.g. Selenium or PhantomJS).
-behat.webdriver_url = http://${behat.webdriver_host}:${behat.webdriver_port}/wd/hub
+behat.webdriver_url = http://localhost:${behat.webdriver_port}/wd/hub
```

Comment:
Why is this reverted? the behat tests are supposed to be run from within the web container. If the tests are run through there, selenium is not installed locally. It needs to connect to the selenium container which is a different one.